src/v2/blobs.rs

```diff
@@ -3,8 +3,11 @@ use futures::Stream;
 use hyper::StatusCode;
 use v2::*;
 
+/// Convenience alias for the binary blob.
+pub type Blob = Vec<u8>;
```

Comment:
What do you plan to use this for? I'm mildly against introducing aliases for plain stdlib types, as it increases the cognitive burden on developers consuming the library.